infrastructure/kubernetes/mezo-staging/monitoring/kustomization.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../common/monitoring
+  - ingress.yaml
+
+patches:
+  - path: metrics-scraper/service-patch.yaml
+  - path: grafana/deployement-patch.yaml
+
+configMapGenerator:
+  - name: grafana-dashboards-mezo
+    files:
+      - grafana/dashboards/mezo-nodes-public.json
+  - name: metrics-scraper-config
+    files:
+      - metrics-scraper/config/config.json
```

Comment:
I really like how `config.json` of the metrics scraper is managed now but I'm afraid we shouldn't use it this way as it directly leaks the (moniker, IP) pairs for monitored nodes. I know this information may be extractable in other ways but we shouldn't make it easier here. I propose we manage `config.json` as a secret created upfront (just as Grafana creds). K8S allows creating a secret directly from a file (see `--from-file` flag of `kubectl create secret`). Let's just make sure we give a good example of how to do so in the README. Last but not least, let's do the same for production or even better - try to generalize it and put in `common/monitoring`.

Reply:
sure thing. I just assumed we would keep the configuration off of the repository, happy with a secret as well.

Reply:
addressed in: 1395e6d260fa907ffca176182da7f1880f65fa12, README as been updated t document this too.